tutorials/interface/activity1.md

```diff
@@ -0,0 +1,242 @@
+
+
+## Your Toolbox
+
+The blocks you need won't always be in the workspace when you start. 
+
+In the instructions, block descriptions will often be highlighted 
+in the same color as the category where they live. As an example, we might use the shorthand ``||game:splash "___"||`` when 
+we want you to find this:
+```block
+game.splash(" ")
+```
+
+<hr/>
+
+**Try:** Let's see how this works. Can you find the ``||scene:set background color to [ ]||`` block and add it into the ``||loops:on start||`` block?
+
+#### ~ tutorialhint 
+```blocks
+scene.setBackgroundColor(0)
+game.splash("My monkey is better than yours")
+```
+
+## The Exception
+
+Every rule has an exception, so let's look at a couple of blocks 
+that don't share the same color as the category where they live.
+
+The ``||variables:set my sprite to [ ]||`` and ``||variables:set my projectile to [ ]||`` blocks
+are both red, but they live inside the ``||sprites:Sprites||`` category.
+
+<hr/>
+
+**Try:** Bring a ``||variables:set my sprite to [ ] of kind [player]||`` into the ``||loops:on start||`` block and 
+play around with it until your [*sprite*](#sprote "A 2-D image that moves on the screen") shows on the screen.
+
+>>*Tip: Drag the ``||game:splash "___"||`` out of the ``||loops:on start||`` block in the workspace
+and drop it back into the toolbox to delete it so that your sprite will be revealed!
+
+#### ~ tutorialhint
+```blocks
+let mySprite = sprites.create(img`
+    e e e . . . . e e e . . . . 
+    c d d c . . c d d c . . . . 
+    c b d d f f d d b c . . . . 
+    c 3 b d d b d b 3 c . . . . 
+    f b 3 d d d d 3 b f . . . . 
+    e d d d d d d d d e . . . . 
+    e d f d d d d f d e . b f b 
+    f d d f d d f d d f . f d f 
+    f b d d b b d d 2 f . f d f 
+    . f 2 2 2 2 2 2 b b f f d f 
+    . f b d d d d d d b b d b f 
+    . f d d d d d b d d f f f . 
+    . f d f f f d f f d f . . . 
+    . f f . . f f . . f f . . . 
+    `, SpriteKind.Player)
+scene.setBackgroundColor(5)
+```
+
+
+## Container Blocks
+
+Now let's look at the different types of blocks and how to use them. 
+
+First, there are [*container blocks*](#blockIt "Blocks that hold other blocks"). 
+Container blocks have an edge at both the the top and bottom with an open space
+in the middle that allows other blocks to snap inside. Container blocks control 
+*when* the code inside runs. Here is an example:
+
+```blocks
+controller.A.onEvent(ControllerButtonEvent.Pressed, function () {
+	
+})
+```
+<hr/>
+**Try:** Find an ``||controller:on [A] button pressed ||`` container 
+block and drag it into the workspace. You can add to it in the next step.
+
+#### ~ tutorialhint
+
+```blocks
+controller.A.onEvent(ControllerButtonEvent.Pressed, function () {
+    let mySprite: Sprite = null
+})
+```
+
+## Standard Blocks
+
+```ghost
+mySprite.startEffect(effects.spray)

```

Comment:
you need to declare mySprite like this:

```
let mySprite: Sprite = null;
```

same with the other snippet. All snippets need to be working bits of code so variables always need to be declared